"use strict";(self.webpackChunkgingras_lab=self.webpackChunkgingras_lab||[]).push([[93],{3176:function(e,t,a){a.r(t),a.d(t,{default:function(){return c}});var n=a(3723),i=a(7294),o=a(1383),l=a(78),r=a(4668),c=function(e){var t=e.data;return i.createElement(l.Z,null,i.createElement(o.Z,{description:"The research interests of Dr. Anne-Claude Gingras and active projects in the lab.",title:"Research"}),i.createElement("div",{className:"research"},i.createElement("section",null,i.createElement("header",null,i.createElement("h1",null,"Research"),i.createElement("img",{alt:"Microscope",height:"25",src:"data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCA1MTIgNTEyIj48IS0tIEZvbnQgQXdlc29tZSBQcm8gNS4xNS4zIGJ5IEBmb250YXdlc29tZSAtIGh0dHBzOi8vZm9udGF3ZXNvbWUuY29tIExpY2Vuc2UgLSBodHRwczovL2ZvbnRhd2Vzb21lLmNvbS9saWNlbnNlIChDb21tZXJjaWFsIExpY2Vuc2UpIC0tPjxkZWZzPjxzdHlsZT4uZmEtcHJpbWFyeXtmaWxsOiNmNjRmNTk7fS5mYS1zZWNvbmRhcnl7ZmlsbDojMTJjMmU5O29wYWNpdHk6LjV9PC9zdHlsZT48L2RlZnM+PHBhdGggZD0iTTEwNCAzODRoMjA4YTggOCAwIDAgMSA4IDh2MTZhOCA4IDAgMCAxLTggOEgxMDRhOCA4IDAgMCAxLTgtOHYtMTZhOCA4IDAgMCAxIDgtOHoiIGNsYXNzPSJmYS1zZWNvbmRhcnkiLz48cGF0aCBkPSJNNDY0IDQ0OGgtMS4yOUExOTEgMTkxIDAgMCAwIDUxMiAzMjBjMC0xMDUuODgtODYuMTItMTkyLTE5Mi0xOTJ2NjRhMTI4IDEyOCAwIDAgMSAwIDI1Nkg0OGE0OCA0OCAwIDAgMC00OCA0OCAxNiAxNiAwIDAgMCAxNiAxNmg0ODBhMTYgMTYgMCAwIDAgMTYtMTYgNDggNDggMCAwIDAtNDgtNDh6TTE2MCAzMjBoMTJ2MTZhMTYgMTYgMCAwIDAgMTYgMTZoNDBhMTYgMTYgMCAwIDAgMTYtMTZ2LTE2aDEyYTMyIDMyIDAgMCAwIDMyLTMyVjY0YTMyIDMyIDAgMCAwLTMyLTMyVjE2YTE2IDE2IDAgMCAwLTE2LTE2aC02NGExNiAxNiAwIDAgMC0xNiAxNnYxNmEzMiAzMiAwIDAgMC0zMiAzMnYyMjRhMzIgMzIgMCAwIDAgMzIgMzJ6IiBjbGFzcz0iZmEtcHJpbWFyeSIvPjwvc3ZnPg=="})),i.createElement("div",{className:"research__links"},i.createElement(r.Z,{nav:!0,to:"/research/#proteomics"},"Proteomics"),i.createElement(r.Z,{nav:!0,to:"/research/#systemsbiology"},"Systems biology"),i.createElement(r.Z,{nav:!0,to:"/research/#signalling"},"Signalling"),i.createElement(r.Z,{nav:!0,to:"/research/#covid19"},"COVID-19 serology"),i.createElement(r.Z,{nav:!0,to:"/research/#funding"},"Funding")),i.createElement("section",{id:"proteomics"},i.createElement("h2",null,"Enabling tools for proteomics"),i.createElement("figure",null,i.createElement("div",{className:"research__image"},i.createElement(n.G,{image:t.proteomics.childImageSharp.gatsbyImageData,alt:"AP-MS and BioID workflows"})),i.createElement("figcaption",null,"AP-MS and BioID workflows. Image courtesy of Kento Abe.")),i.createElement("p",null,"We are developing experimental and computational pipelines for proteomics, with a focus on interaction and affinity workflows. Experimentally, we have developed new sets of lentiviral vectors to express BioID fusions across multiple cell types (",i.createElement(r.Z,{to:"https://www.mcponline.org/article/S1535-9476(20)32027-2/fulltext"},"Samavarchi-Tehrani,"," ",i.createElement("span",{className:"italic"},"MCP")," ","2018"),"), and protocols to perform proximity-dependent biotinylation in zebrafish embryos (",i.createElement(r.Z,{to:"https://www.mcponline.org/article/S1535-9476(21)00100-6/fulltext"},"Rosenthal,"," ",i.createElement("span",{className:"italic"},"MCP")," ","2021"),"; with Ian Scott). Our current projects explore combining proximity-dependent biotinylation with"," ",i.createElement("span",{className:"italic"},"in vivo")," ","crosslinking, looking at context-dependency with split enzymes, and developing approaches to perform proximity-dependent biotinylation extracellularly. Our computational tools range from a LIMS for proteomics (",i.createElement(r.Z,{to:"https://www.nature.com/articles/nbt1010-1015"},"ProHits: Liu,"," ",i.createElement("span",{className:"italic"},"Nature Biotechnology")," ","2010"),") to tools for scoring protein-protein interactions (",i.createElement(r.Z,{to:"https://www.nature.com/articles/nmeth.1541"},"SAINT: Choi,"," ",i.createElement("span",{className:"italic"},"Nature Methods")," ","2011"),";"," ",i.createElement(r.Z,{to:"https://www.nature.com/articles/nmeth.2557"},"CRAPome: Mellacheruvu,"," ",i.createElement("span",{className:"italic"},"Nature Methods")," ","2013"),"), co-developed with Alexey Nesvizhskii, Hyungwon Choi and Mike Tyers. We also developed tools to visualize and further analyze interactions, most notably"," ",i.createElement(r.Z,{to:"https://www.nature.com/articles/nmeth.4330"},"ProHits-viz (Knight,"," ",i.createElement("span",{className:"italic"},"Nature Methods")," ","2017)"),", and introduced"," ",i.createElement(r.Z,{to:"https://www.nature.com/articles/s41592-019-0477-9"},"GIX  (Knight,"," ",i.createElement("span",{className:"italic"},"Nature Methods")," ","2019)"),", a browser extension for rapid annotation of gene products. With Alexey Nesvizhskii, Nuno Bandeira and Hannes Röst, we have contributed to the development of tools for the interpretation of Data Independent Acquisition results. Current efforts aim to improve the connectivity between all these computational tools.")),i.createElement("section",{id:"systemsbiology"},i.createElement("h2",null,"Systems biology"),i.createElement("figure",null,i.createElement("div",{className:"research__image"},i.createElement(n.G,{image:t.systemsBiology.childImageSharp.gatsbyImageData,alt:"Cartoon of a human cell with labelled organelles used for mapping"})),i.createElement("figcaption",null,"Targeted compartments for mapping the cell. Image courtesy of Kento Abe.")),i.createElement("p",null,"We have long taken a protein family-centric approach to explore specificity in the interactions established by kinases (",i.createElement(r.Z,{to:"https://science.sciencemag.org/content/328/5981/1043.long"},"Breitkreutz,"," ",i.createElement("span",{className:"italic"},"Science")," ","2010"),"; with Mike Tyers and Alexey Nesvizhskii), phosphatases (",i.createElement(r.Z,{to:"https://www.cell.com/cell-reports/fulltext/S2211-1247(16)31513-3"},"St-Denis,"," ",i.createElement("span",{className:"italic"},"Cell Reports")," ","2016"),"), bromodomain-containing proteins (",i.createElement(r.Z,{to:"https://www.cell.com/molecular-cell/fulltext/S1097-2765(18)30948-1"},"Lambert,"," ",i.createElement("span",{className:"italic"},"Molecular Cell")," ","2018"),"), and chaperones (",i.createElement(r.Z,{to:"https://www.cell.com/cell/fulltext/S0092-8674(14)00740-5"},"Taipale,"," ",i.createElement("span",{className:"italic"},"Cell")," ","2014"),";"," ",i.createElement(r.Z,{to:"https://www.sciencedirect.com/science/article/abs/pii/S1097276521003178"},"Piette,"," ",i.createElement("span",{className:"italic"},"Molecular Cell")," ","2021"),"). The introduction of proximity-dependent biotinylation in 2012 has now enabled us to provide an organelle-specific view of the proteome. We have notably described the proteome of cytosolic RNA-containing granules and bodies using BioID (",i.createElement(r.Z,{to:"https://www.cell.com/molecular-cell/fulltext/S1097-2765(17)30977-2"},"Youn,"," ",i.createElement("span",{className:"italic"},"Molecular Cell")," ","2018"),";"," ",i.createElement(r.Z,{to:"https://www.cell.com/molecular-cell/fulltext/S1097-2765(19)30699-9"},"Youn,"," ",i.createElement("span",{className:"italic"},"Molecular Cell")," ","2019"),"), and more recently, in collaboration with Eric Shoubridge, the organization of the human mitochondria (",i.createElement(r.Z,{to:"https://www.cell.com/cell-metabolism/fulltext/S1550-4131(20)30412-5"},"Antonicka,"," ",i.createElement("span",{className:"italic"},"Cell Metabolism")," ","2020"),"). Lessons learned from these organelle-specific studies have enabled the establishment of a comprehensive proximity map of a human cell (",i.createElement(r.Z,{to:"https://www.nature.com/articles/s41586-021-03592-2"},"Go, Knight,"," ",i.createElement("span",{className:"italic"},"Nature")," ","2021"),") and a dedicated website that enables user to navigate our results and analyze their own BioID experiments (",i.createElement(r.Z,{to:"https://humancellmap.org"},"humancellmap.org"),"). We have also repurposed the humancellmap.org to help provide context to the proximity interactomes of SARS-CoV-2 proteins (",i.createElement(r.Z,{to:"https://www.biorxiv.org/content/10.1101/2020.09.03.282103v1"},"Samavarchi-Tehrani,"," ",i.createElement("span",{className:"italic"},"bioRχiv")," ","2021"),";"," ",i.createElement(r.Z,{to:"https://covid19interactome.org"},"covid19interactome.org"),"). Some of the current projects in the lab involve high-resolution maps of nuclear bodies, exploration of dynamic changes in the proximal proteomes following induction of somatic cell reprogramming, and studying the consequences of alternative splicing on interactomes.")),i.createElement("section",{id:"signalling"},i.createElement("h2",null,"Signalling"),i.createElement("figure",null,i.createElement("div",{className:"research__image"},i.createElement(n.G,{image:t.signalling.childImageSharp.gatsbyImageData,alt:"MTOR signalling at the lysosome membrane"})),i.createElement("figcaption",null,"MTOR signalling at the lysosome membrane. Image courtesy of Geoff Hesketh.")),i.createElement("p",null,"Some of our core research interests have been the study of signaling pathways that are deregulated in cancers and rare diseases. In 2009, we identified the STRIPAK complex (STRiatin-Interacting Phosphatase And Kinase;"," ",i.createElement(r.Z,{to:"https://www.mcponline.org/article/S1535-9476(20)30850-1/abstract"},"Goudreault,"," ",i.createElement("span",{className:"italic"},"MCP")," ","2009"),") that plays important roles in the Hippo signaling pathway (",i.createElement(r.Z,{to:"https://stke.sciencemag.org/content/6/302/rs15"},"Couzens,"," ",i.createElement("span",{className:"italic"},"Science Signaling")," ","2013"),") and in Cerebral Cavernous Malformations, a vascular disease. In collaboration with the Sicheri lab, we have defined the phospho-dependent recruitment of proteins in the core kinase cascade of the Hippo pathway (",i.createElement(r.Z,{to:"https://www.mcponline.org/article/S1535-9476(20)32373-2/fulltext"},"Xiong,"," ",i.createElement("span",{className:"italic"},"MCP")," ","2017"),";"," ",i.createElement(r.Z,{to:"https://www.mcponline.org/article/S1535-9476(20)32372-0/fulltext"},"Couzens,"," ",i.createElement("span",{className:"italic"},"MCP")," ","2017"),"), and have helped our Terry Fox Program co-investigators to identify additional components in this pathway. Our efforts on the context-dependent aspects of Cerebral Cavernous Malformation signaling will make use of our newly developed"," ",i.createElement("span",{className:"italic"},"in vivo")," ","zebrafish biotinylation system. Through the use of proximity-dependent biotinylation sensors, we have uncovered that the free-amino acid sensing pathway for mTORC1 activation (the GATOR-Rag GTPase pathway) actively suppresses an alternative pathway that activates mTORC1 through amino acids produced from lysosomal degradation of proteins (",i.createElement(r.Z,{to:"https://science.sciencemag.org/content/370/6514/351.long"},"Hesketh,"," ",i.createElement("span",{className:"italic"},"Science")," ","2020"),"). Future work in this area includes a more comprehensive mapping of the mTORC1 network, as well as continued efforts, some with the labs of Jean-François Côté and Matthew Smith, in characterizing proximity networks for small GTPases. Additionally, we are also exploring receptor kinase signaling using proximity-dependent biotinylation, looking at both intracellular and extracellular interactions.")),i.createElement("section",{id:"covid19"},i.createElement("h2",null,"COVID-19 serology"),i.createElement("figure",null,i.createElement("div",{className:"research__image"},i.createElement(n.G,{image:t.serology.childImageSharp.gatsbyImageData,alt:"COVID-19 serology workflow"})),i.createElement("figcaption",null,"Serology workflows. Image courtesy of Kento Abe.")),i.createElement("p",null,"While the focus of the lab remains functional proteomics, systems biology and signal transduction, our expertise in protein-protein interactions and our affiliation with the"," ",i.createElement(r.Z,{to:"https://nbcc.lunenfeld.ca"},"Network Biology Collaborative Centre (NBCC)")," ","placed us in an excellent position to develop antibody-based assays for COVID-19 in March 2020. We received early funding through the"," ",i.createElement(r.Z,{to:"https://supportsinai.ca"},"Sinai Foundation"),", from the"," ",i.createElement(r.Z,{to:"https://www.krembilfoundation.ca"},"Krembil Foundation")," ","and the"," ",i.createElement(r.Z,{to:"https://www.rbc.com/community-social-impact/"},"RBC Foundation")," ","that enabled us to build assays. With Jim Rini, we developed a protein-based neutralization surrogate for SARS-CoV-2 (",i.createElement(r.Z,{to:"https://insight.jci.org/articles/view/142362"},"Abe,"," ",i.createElement("span",{className:"italic"},"JCI Insight")," ","2020"),") and re-implemented a lentiviral neutralization surrogate assay introduced by Jesse Bloom. We partnered with colleagues at the"," ",i.createElement(r.Z,{to:"https://nrc.canada.ca"},"National Research Council of Canada (NRC)"),", the"," ",i.createElement(r.Z,{to:"https://www.canada.ca/en/public-health/programs/national-microbiology-laboratory.html"},"National Microbiology Laboratory (NML)"),", colleagues in the"," ",i.createElement(r.Z,{to:"http://www.tibdn.ca"},"Toronto Invasive Bacterial Diseases Network (TIBDN)")," ","and the"," ",i.createElement(r.Z,{to:"https://www.blood.ca"},"Canadian Blood Services")," ","to develop high-throughput serology assays at the NBCC. These assays have been developed by a dedicated team, co-led by Anne-Claude and NBCC manager Karen Colwill. The assays have been used notably to demonstrate, in a collaboration with Jen Gommerman and Allison McGeer, that anti-spike antibodies in the blood and saliva of infected individuals lasted for at least 3-4 months (the latest time point collected in that particular study;"," ",i.createElement(r.Z,{to:"https://immunology.sciencemag.org/content/5/52/eabe5511.long"},"Isho,"," ",i.createElement("span",{className:"italic"},"Science Immunology")," ","2020"),"). The assays are now used for >40 different projects across the country that monitor seroprevalence or antibody levels following infection or vaccination, many funded through the"," ",i.createElement(r.Z,{to:"https://cihr-irsc.gc.ca"},"Canadian Institutes of Health Research (CIHR)")," ","and the"," ",i.createElement(r.Z,{to:"https://www.covid19immunitytaskforce.ca"},"COVID-19 Immunity Task Force (CITF)"),". Several of the studies in vaccinated populations focus on individuals with compromised immune systems, including residents of long-term care homes (with Sharon Straus and Allison McGeer), patients receiving dialysis (with Michelle Hladunewich), older individuals (with Sharon Walmsley), and patients with autoimmune disease (with Vinod Chandran, Tania Watts, Mark Silverberg, Sasha Bernatsky and Dawn Bowdish). We also contribute to development of new therapeutics and vaccines (see, e.g."," ",i.createElement(r.Z,{to:"https://www.science.org/doi/10.1126/sciadv.abj9815"},"Liu,"," ",i.createElement("span",{className:"italic"},"Science Advances")," ","2021"),"). Lastly, Anne-Claude leads the Pillar 3: Functional Genomics and Structure/Function group for the newly funded CIHR network for Variants of Concern (CoVaRR-Net), led by Marc-André Langlois at the University of Ottawa.")),i.createElement("section",{id:"funding"},i.createElement("h2",null,"Funding"),i.createElement("p",null,"We are grateful to the following agencies for supporting the projects in the Gingras lab and the LTRI proteomics group:"),i.createElement("ul",null,i.createElement("li",null,i.createElement(r.Z,{to:"http://www.cihr-irsc.gc.ca"},"Canadian Institutes of Health Research")),i.createElement("li",null,i.createElement(r.Z,{to:"http://www.nserc-crsng.gc.ca"},"Natural Sciences and Engineering Research Council")),i.createElement("li",null,i.createElement(r.Z,{to:"https://www.cancer.ca/research"},"Canadian Cancer Society Research Institute")),i.createElement("li",null,i.createElement(r.Z,{to:"https://www.crs-src.ca/"},"Cancer Research Society")),i.createElement("li",null,i.createElement(r.Z,{to:"https://www.genomecanada.ca"},"Genome Canada")),i.createElement("li",null,i.createElement(r.Z,{to:"https://www.ontariogenomics.ca/"},"Ontario Genomics Institute")),i.createElement("li",null,i.createElement(r.Z,{to:"https://www.innovation.ca/"},"Canada Foundation for Innovation")),i.createElement("li",null,i.createElement(r.Z,{to:"https://www.nih.gov/"},"National Institutes of Health")),i.createElement("li",null,i.createElement(r.Z,{to:"http://www.cihr-irsc.gc.ca/e/13147.html"},"CIHR Institute of Genetics")),i.createElement("li",null,i.createElement(r.Z,{to:"https://www.computecanada.ca/"},"Compute Canada")),i.createElement("li",null,i.createElement(r.Z,{to:"http://www.chairs-chaires.gc.ca/home-accueil-eng.aspx"},"Canada Research Chair")))))))}}}]);